src/WebJobs.Script/Config/ScriptHostConfiguration.cs

```diff
@@ -50,6 +51,14 @@ public ScriptHostConfiguration()
         /// The default is false. When set to true, logs will be written to the directory
         /// specified by <see cref="RootLogPath"/>.
         /// </summary>
-        public bool FileLoggingEnabled { get; set; }    
+        public bool FileLoggingEnabled { get; set; }
+
+        /// <summary>
+        /// Gets the list of functions that should be run. This list can be used to filter
+        /// the set of functions that will be enabled - it can be a subset of the actual
+        /// function directories. When left null (the default) all discovered functions will
+        /// be run.
+        /// </summary>
+        public Collection<string> Functions { get; set; }
```

Comment:
Yeah, and I'll likely give it more thought anyways, I'll likely evolve this over time. It is definitely very useful for local testing. Give it a try and let me know. I don't intend this as an official feature.